Orange/widgets/data/tests/test_owtranspose.py

```diff
@@ -33,7 +33,30 @@ def test_parameters(self):
         self.assertListEqual(
             [a.name for a in self.get_output(self.widget.Outputs.data).domain.attributes],
             [self.zoo.domain.metas[0].to_val(m) for m in self.zoo.metas[:, 0]])
-        self.widget.feature_radio.buttons[0].click()
+        self.widget.feature_type = self.widget.GENERIC
         self.widget.apply()
         self.assertTrue(all(["Feature" in x.name for x
                              in self.get_output(self.widget.Outputs.data).domain.attributes]))
+
+    def test_transpose_custom_name(self):
+        """
+        Custom name can be set.
+        GH-2737
+        """
+        w = self.widget
+        self.send_signal(w.Inputs.data, self.zoo)
+        self.assertEqual(w.feature_type, w.FROM_META_ATTR)
+        w.controls.feature_type.buttons[w.GENERIC].click()
+        self.assertEqual(w.feature_type, w.GENERIC)
+        CUSTOM_TEXT = "Bravo"
+        w.feature_name = CUSTOM_TEXT
+        w.apply()
+        self.assertTrue(all([CUSTOM_TEXT in v.name for v
+                             in self.get_output(w.Outputs.data).domain.attributes]))
+
+    def test_send_report(self):
+        """
+        GH-2773
```

Comment:
Here is an even better example: `test_send_report` should be a test for reports. Why is it marked as "GH-2737"?